components/CompletionScoreCard.tsx

```diff
@@ -0,0 +1,213 @@
+
+const mockData = {
+  title: 'Zero COventry',
+  linkText: 'netzerocities.kausal.tech/coventry-netzero',
+  linkHref: '#',
+  scoreData: [
+    { status: 'High', total: 13, completed: 10 } as scoreData,
+    { status: 'Medium', total: 10, completed: 7 } as scoreData,
+    { status: 'Low', total: 8, completed: 2 } as scoreData,
+  ],
+};
+
+const calculateScores = (scoreData: scoreData[]) => {
+  return scoreData.map((data) => ({
+    status: data.status,
+    score: `${data.completed}/${data.total}`,
+  }));
+};
+
+const calculatePercentage = (scoreData: scoreData[]) => {
+  const totalCompleted = scoreData.reduce(
+    (acc, curr) => acc + curr.completed,
+    0
+  );
+  const totalItems = scoreData.reduce((acc, curr) => acc + curr.total, 0);
+  return totalItems === 0 ? 0 : (totalCompleted / totalItems) * 100;
+};
+
+export const CompletionScoreCard: React.FC<CompletionScoreCardProps> = () => {
+  const [data, setData] = useState<MockData | null>(null);
+  const [scores, setScores] = useState<
+    { status: 'High' | 'Medium' | 'Low'; score: string }[]
+  >([]);
+  const [completionPercentage, setCompletionPercentage] = useState<number>(0);
+
+  useEffect(() => {
+    setTimeout(() => {
+      setData(mockData);
+      const calculatedScores = calculateScores(mockData.scoreData);
+      setScores(calculatedScores);
+      const percentage = calculatePercentage(mockData.scoreData);
+      setCompletionPercentage(percentage);
+    }, 1000);
+  }, []);
+
+if (!data) {
+  return <Typography>Loading...</Typography>;
+}
+
+  return (
+    <Box
+      sx={{
+        width: '80%',
+        margin: '0 auto',
+        marginTop: '7em',
+        border: '1px solid #ddd',
+        borderRadius: '0.5em',
+        padding: '1em',
+        backgroundColor: '#f9f9f9',
+        boxShadow: '0 4px 8px rgba(0, 0, 0, 0.1)',
+      }}
+    >
+      <Box
+        sx={{
+          display: 'flex',
+          justifyContent: 'space-between',
+          alignItems: 'center',
+          marginBottom: '1em',
+        }}
+      >
+        <Box>
+          <Typography variant="h5">{data.title}</Typography>
+          <Box
+            component="span"
+            sx={{
+              display: 'inline-flex',
+              alignItems: 'center',
+              textDecoration: 'none',
+              '&:hover': {
+                borderBottom: '1px solid',
+              },
+            }}
+          >
+            <Link
+              href={data.linkHref}
+              variant="body2"
+              sx={{
+                display: 'inline-flex',
+                alignItems: 'center',
+                textDecoration: 'none',
+              }}
+            >
+              {data.linkText}
+              <ArrowUpwardIcon
+                sx={{
+                  fontSize: '1.1em',
+                  transform: 'rotate(45deg)',
+                  marginLeft: '0.3em',
+                }}
+              />
+            </Link>
+          </Box>
+        </Box>
+        <Button
```

Comment:
I _think_ we should add `component={Link}` here for semantics (so it renders an `<a>` rather than `<button>`)